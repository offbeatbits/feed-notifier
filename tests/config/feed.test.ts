describe("Feed config", () => {
  it("should export feed config", () => {
    expect(require("../../src/config/feed")).toMatchSnapshot(); // eslint-disable-line
  });
});
