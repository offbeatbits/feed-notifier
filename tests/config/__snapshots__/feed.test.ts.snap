// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed config should export feed config 1`] = `
Object {
  "feedUrl": "https://example.org/test_feed.xml",
}
`;
